Bugfix/2.4 groups

ericmartin opened the pull request:
There's a bug when updating non-empty IP and domain groups, since the VAT is looking for a list of IDs as opposed to free strings. This separates out the behavior based on group type. The changes are to **update_group()**

ericmartin committed:
VSCode formatting updates

ericmartin committed:
fixed update_group()

update_group() would fail with an error when updating non-empty
domain and IP based groups.

HoneyBaked commented on .gitignore:
```diff
@@ -0,0 +1,5 @@
+*.pyc
+.coverage
+*,cover
```
Should this be: *.cover

ericmartin replied:
The [pytest-cov](https://pypi.org/project/pytest-cov/) module creates a file named **file.py,cover** when you request code annotations `coverage annotate vectra.py.` That being said, I just looked and the functionality is being deprecated in favor of HTML reports which go in the **htmlcov** directory.

I'll remove this line and add the **htmlcov** line

ericmartin committed:
changes to .gitignore

pytest-cov no longer supports inline annotations (*,cover).
Swapped to htmlcov for html reports
